packages/react-form-renderer/src/validators/validator-functions.js

```diff
@@ -90,7 +90,7 @@ export const numericality = memoize(
     lessOrEqual = selectNum(lessOrEqual, lessThanOrEqualTo);
 
     return prepare((value) => {
-      if (!value) {
+      if (value == null) {
```

Comment:
I think we should cover all "falsy" values
``` suggestion
      if (typeof value !== 'number' && typeof value !== 'string') {
```

Reply:
The next block to this is checking if the value is a valid number or not, handling these here will lead to no proper message being displayed.

Reply:
You are correct. My point is that we also have to consider `undefined` in addition to `null`. I guess we can let the rest of the types through this check.

Reply:
I have not added strict equality intentionally, to cover "null" as well as "undefined" case.

Reply:
These strings will be handled by the `isNumber` check below as you pointed out. We like to use the strict equality operator whenever possible to avoid unexpected "truthy" results. It's a matter of convention.

We can go ahead and merge/release the PR afterward.

Reply:
Sorry if I was not able to explain it properly. I didn't meant to cover the null and undefined "strings" but the values, so basically to cover (value === null || value === undefined) with just ( value == null ) as it will return true for undefined also. Let me know if you think we should explicitly check for undefined with strict equality, I will make the change.

Reply:
@jspannu919 yes, please. We try to use strict equality as much as possible to avoid some unintentional condition outcomes.

Reply:
Done